src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -574,7 +575,7 @@ void SpatialPooler::initialize(vector<UInt> inputDimensions,
 
   tieBreaker_.resize(numColumns_);
   for (UInt i = 0; i < numColumns_; i++) {
-    tieBreaker_[i] = 0.01 * rng_.getReal64();
+    tieBreaker_[i] = pow(10, -1*(getPrecision()+1)) * rng_.getReal64(); // (PRECISION+1)-th place
```

Comment:
This is the source of unit test failure for me.  If I replace this line with, say, `tieBreaker_[i] = rng_.getReal64() / pow(10, getPrecision()+1);`, which is the equivalent of the _intent_, it works for me.  But if I leave it as is, I get `inf` for the tie breakers, which I think is not intended.